src/test/scala/fi/oph/koski/api/SuostumuksenPeruutusSpec.scala

```diff
@@ -35,6 +37,11 @@ class SuostumuksenPeruutusSpec extends AnyFreeSpec with Matchers with Opiskeluoi
       }
     }
 
+    "Opiskeluoikeus on poistunut Elasticsearchista" in {
```

Comment:
Tämä testi ei taida olla ajettavissa itsenäisenä? Testit pitäisi koodata niin, eikä riippuvaisiksi toisistaan.

Reply:
Ihan kaikissa testeissä näin ei ole ollut, esim SuoritusJakoSpec

Minusta saman testifilun sisällä testien toimiminen toisistaan riippumattomina ei tunnu tarpeelliselta. Tulloo aika paljon ylimääräistä koodia ja/tai ylimääräistä ajoaikaa jos pitää aina peruuttaa muutokset jokaisen saman kokonaisuuden testin jälkeen.